schema/artifacts.hcl

```diff
@@ -69,8 +79,8 @@ table "artifacts" {
     on_update   = NO_ACTION
     on_delete   = NO_ACTION
   }
-  foreign_key "artifacts_playbook_run_fkey" {
-    columns     = [column.playbook_run_id]
+  foreign_key "artifacts_playbook_run_action_fkey" {
+    columns     = [column.playbook_run_action_id]
     ref_columns = [table.playbook_runs.column.id]
```

Comment:
The table would be `playbook_run_actions`

Reply:
👍🏽